configs/common/modules.yaml

```diff
@@ -334,6 +336,8 @@ modules:
         suffixes:
           ^esmf@8.2.0~debug: 'esmf-8.2.0'
           ^esmf@8.2.0+debug: 'esmf-8.2.0-debug'
+          ^esmf@8.3.0b09~debug: 'esmf-8.3.0b09'
+          ^esmf@8.3.0b09+debug: 'esmf-8.3.0b09-debug'
```

Comment:
I have an improvement for this that will allow us to generically use versions instead of listing them manually.